src/FactorAPI/ComputeEngineIntegrationTest.java

```diff
@@ -0,0 +1,29 @@
+package factorapi;
+
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ComputeEngineIntegrationTest {
+    @Test
+    void testComputeEngineIntegration() {
+        List<Integer> inputList = List.of(1, 10, 25);
+        List<String> outputList = new ArrayList<>();
+
+        InMemoryInputConfig inputConfig = new InMemoryInputConfig(inputList);
+        InMemoryOutputConfig outputConfig = new InMemoryOutputConfig(outputList);
+        InMemoryDataStore dataStore = new InMemoryDataStore(inputConfig, outputConfig);
+        
```

Comment:
I think maybe it's better if this only has an object for the dataStore. It could pass it the inputConfig through the constructor.